apps/portal/controllers/error-pages/error401.html

```diff
@@ -17,7 +33,20 @@
 <i class="icon-unlink error-icon"></i>
 <h1>Error 401</h1>
 <h3>You do not have permission to access this page.Please contact your administrator and request permission.</h3>
-<h4><a href="/portal">Go to Portal</a></h4>
+<div id="portal" style="visibility: visible; display: block; ..">
+    <h4><a href="/portal">Go to Portal</a></h4>
+</div>
+<script type="text/javascript">
+    var link = document.getElementById('portal');
```

Comment:
better be consistent with quotations for defining strings, use double quotes for outer strings and single quotes for inner strings.